test/functional/mempool_package_limits.py

```diff
@@ -177,6 +178,107 @@ def test_desc_count_limits(self):
         node.generate(1)
         assert all([res["allowed"] for res in node.testmempoolaccept(rawtxs=package_hex)])
 
+    def test_desc_count_limits_2(self):
+        """Create A bushy Package with 24 transaction in mempool and 3 transaction in package
+                              M1a
+                             ^  ^
+                            ^    M2a
+            M1b            ^       .
+              ^           ^         .
+               ^         ^           ^
+                ^       ^            M23a
+                 ^     ^
+                  ^   P1
+                   ^ ^  ^
+                    P2   P3
```

Comment:
nit: this test would be simplified if you made M2a .... M23a be a chain spending vout[0] of M1a. It'd save maybe 5-10 lines.